index.html

```diff
@@ -85,7 +85,7 @@ <h2>Examples</h2>
 									<a href="examples/backbone/" data-source="http://backbonejs.org/" data-content="Backbone.js gives structure to web applications by providing models with key-value binding and custom events, collections with a rich API of enumerable functions, views with declarative event handling, and connects it all to your existing API over a RESTful JSON interface.">Backbone.js</a>
 								</li>
 								<li class="routing">
-									<a href="examples/angularjs/" data-source="http://angularjs.org" data-content="What HTML would have been had it been designed for web apps">AngularJS</a>
+									<a href="updates/angular/dist/" data-source="https://angular.io/" data-content="Angular is a development platform for building mobile and desktop web applications using TypeScript/JavaScript and other languages.">Angular</a>
```

Comment:
I'd propose using the content from angular.dev (new website):
```suggestion
									<a href="updates/angular/dist/" data-source="https://angular.dev" data-content="Angular is a web framework that empowers developers to build fast, reliable applications.">Angular</a>
```
The proposed text was taken from https://angular.dev/overview.